### DISCUSSION TITLE
Are there any ready-made linux binary packages available for download?

### DISCUSSION THREAD

**Discussion by KingsonKai** [eunomia-bpf/wasm-bpf#139] [Q&A]
When I tried to install wasm-bpf using the "cargo install --locked wasm-bpf" command, an error occurred.
<img width="997" alt="image" src="https://github.com/eunomia-bpf/wasm-bpf/assets/27922190/35b2ec85-6896-4c39-9d06-048a72ef0670">

**Comment by oluceps** [comment]
Here is an easier way `nix build github:eunomia-bpf/wasm-bpf`
